core/app/models/spree/return_item/eligibility_validator/default.rb

```diff
@@ -1,5 +1,3 @@
-require_relative 'r_m_a_required'
```

Comment:
I noticed that `require_relative` does the trick for file named `rma_required` (then adding acronym is not neccessary)